XRTK-Core/Packages/com.xrtk.core/Definitions/Devices/MixedRealityInteractionMapping.cs

```diff
@@ -193,6 +258,15 @@ public MixedRealityInputAction MixedRealityInputAction
         /// </summary>
         public KeyCode KeyCode => keyCode;
 
+        [SerializeField]
+        [Tooltip("Optional KeyCode value to get input from Unity's old input system.")]
+        private string inputName;
+
+        /// <summary>
+        /// Optional inputName value to get input for a coded input identity from a provider.
```

Comment:
Shouldn't the summary match the tooltip description?

Reply:
Will double check